main.gms

```diff
@@ -685,7 +685,7 @@ parameter
   cm_prtpScen               "pure rate of time preference standard values"
 ;
   cm_prtpScen         = 1;         !! def = 1  !! regexp = 1|3
-*' *  (1): 1 %
```

Comment:
I am confuesed...isn't it 1 in the line above?

Reply:
That had me confused too, but I think it is supposed to be just an index for the switch options. Because later it clearly says
```
*** define pm_prtp according to cm_prtpScen:
if(cm_prtpScen eq 1, pm_prtp(regi) = 0.015);
if(cm_prtpScen eq 3, pm_prtp(regi) = 0.03);
```
But I guess we could also just make this switch into a numerical one, so the switch value actually corresponds to the value of the pure rate of time preference. That would also a nice development feature if one wants to vary the rate.

Reply:
ah, thanks. Let's keep it as is